adobe/react-spectrum
# docs: add rss feed to release notes and blog posts

Closes https://github.com/adobe/react-spectrum/issues/2735

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Go to docs (https://reactspectrum.blob.core.windows.net/reactspectrum/07abf1452d9a9e1b78df0997c54fe1b570bdfef1/verdaccio/docs/index.html)
2. Go to either our release notes or blog posts and scroll to the button
3. Click on the RSS Link which should open a new tab
4. If you want, you can try subscribing to that url in some aggregator and see how it looks

## 🧢 Your Project:
